### PR TITLE
new package: py-jdatetime

### PR DESCRIPTION
**Pull request by healther**

### CODE REVIEW

**Comment by adamjstewart** [comment]
A lot of unit tests are failing or timing out. This is obviously unrelated to this PR. I've tried restarting them several times but to no avail. I'm going to try closing and reopening. If that doesn't work, we can try a rebase.

**Comment by adamjstewart** [comment]
Nope, same problem. Can you try rebasing?

**Comment by healther** [comment]
done

File: var/spack/repos/builtin/packages/py-jdatetime/package.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyJdatetime(PythonPackage):
+    """jdatetime is Jalali implementation of Python’s datetime module"""
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-jdatetime/package.py]: This is a long shot, but maybe the smart quote in this docstring is what is causing the problem? That would explain why only Python 2 is affected and not Python 3. Try changing this to a regular quote.
```suggestion
    """jdatetime is Jalali implementation of Python's datetime module"""
```

**Pull request merged**